docs/contributing-to-analyses/analysis-modules/notebook-structure.md

```diff
@@ -143,10 +144,120 @@ The Markdown text should focus on providing a higher-level overview of the analy
 
 ### Session info
 
-The final section of every notebook should be a **Session info** section that uses the [`sessionInfo()` function](../determining-requirements/determining-software-requirements.md#using-sessioninfo) to print out the versions of all packages used in the analysis.
+The final section of every R Markdown notebook should be a **Session info** section that uses the [`sessionInfo()` function](../determining-requirements/determining-software-requirements.md#using-sessioninfo) to print out the versions of all packages used in the analysis.
 
 !!! tip
     In `hello.Rmd`, we use [the `sessioninfo` package](https://sessioninfo.r-lib.org) to generate a slightly nicer output format using the `sessioninfo::session_info()` function; either approach is fine!
 
 
 ## Jupyter notebooks
+
+We will illustrate the structure of a Jupyter notebook using the example notebook found in the [`hello-python`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/hello-python) analysis module: [`hello.ipynb`](https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/main/analyses/hello-python/hello.ipynb).
+
+
+### Headers
+
+Jupyter notebooks do not have a YAML header like R Markdown notebooks, but you should still begin with a Markdown cell at the top of the notebook that includes a title, author, and date, as shown in this example:
+
+```markdown
+# Hello OpenScPCA
+
+Childhood Cancer Data Lab, ALSF
+
+2024-03-19
+```
+
+The date should be kept current as the notebook is modified.
```

Comment:
What do you think about recommending `datetime`? I guess this would have to be in a separate chunk, so I don't really love it.

```
from datetime import date
today = date.today() 
print(today)
```

Reply:
Yeah, that is why I didn't do that.